phpBB/includes/mcp/mcp_post.php

```diff
@@ -197,7 +213,7 @@ function mcp_post_details($id, $mode, $action)
 		$l_deleted_by = '';
 	}
 
-	$template->assign_vars(array(
+	$mcp_post_template_data = array(
```

Comment:
Not sure if this needs changing. You can simply overwrite them. but I guess it doesn't hurt either